src/test/java/org/icgc_argo/workflow/relay/util/Fixture.java

```diff
@@ -20,6 +23,11 @@
       Class clazz, String fileName, Class<T> targetClass, ObjectMapper customMapper) {
     String json = loadJsonString(clazz, fileName);
     customMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+
+    val module = new SimpleModule();
+    module.addDeserializer(OffsetDateTime.class, new OffsetDateTimeDeserializer());
```

Comment:
Since we are creating these modules in a couple places now, we could we add a static method inside the deserializer that returns a module ...

```
customMapper.registerModule(OffsetDateTimeDeserializer.getModule())
```